server/app/services/geo/esri/RealEsriClient.java

```diff
@@ -37,6 +38,13 @@
 public final class RealEsriClient extends EsriClient implements WSBodyReadables, WSBodyWritables {
   private final WSClient ws;
 
+  private static final Counter esriRequestCount =
```

Comment:
nitpick: use SCREAMING_SNAKE_CASE for constants

Reply:
Done